frontend/src/components/modals/Settings.tsx

```diff
@@ -68,6 +68,7 @@ export default function Settings({ openRef }: { openRef: RefObject<HTMLButtonEle
                             GlobalOptions.EnsureAlwaysFiveCubitShopOptions,
                             GlobalOptions.EnableFreeCubitShop,
                             GlobalOptions.UnlockAllCostumes,
+                            GlobalOptions.SlideToCWS,

```

Comment:
This is just a personal opinion, so feel free to ignore, but I think avoiding the abbreviation might be clearer, i.e. `SlideToCoWorkingSpace`